netpyne/cell.py

```diff
@@ -525,7 +525,7 @@ def _addConnPlasticity (self, params, netcon, weightIndex):
         plasticity = params.get('plasticity')
         if plasticity and sim.cfg['createNEURONObj']:
             try:
-                plastSection = h.Section()
+                plastSection = self.secs['soma']['hSection']
```

Comment:
this assumes there is a 'soma' section -- I'll make change in my branch to make more generic